### PR TITLE
Update package/assembly versions for assemblies to be serviced 1.0

### LINKED ISSUE #23119: Update package versions to service release/1.1 and 1.0 branches.
I went through the history of the release/1.1 branch, this [commit](https://github.com/dotnet/corefx/commit/9e98f24f508d2cda97f10474a86052f51bc9c7e3), which is an update after the servicing release. From then these are the concrete product commits,

1. [Interop.MountPoints.cs](https://github.com/dotnet/corefx/commit/3d2debcf3f4fb7ce06f58c0b617348835e9036c8) by @wfurt 
2. [WinHttpHandler](https://github.com/dotnet/corefx/commit/454bdeb2ef477344b8626307892f442bff2a3106) by @davidsh 

We need to update the package and assembly versions of the libs where these changes need to serviced for 1.1 release.

refer pr #23113

### PR DESCRIPTION
**Pull request by Priya91**
cc @danmosemsft @ericstj 

fixes #23119 

@ericstj The System.IO.FileSystem.DriveInfo pkgproj doesn't have the last package version, and looking at nuget.org, the package version seems to be 4.3.0, but not sure which branch version is that. How do I update the package version of this lib here?

### CODE REVIEW

**Comment by bartonjs** [comment]
4.3.0 is the 1.1 release; the 1.0 release built 4.0.0, so you'd bump it to 4.0.1 here by manually adding the Version property to the `pkgproj` file.

(Apparently in release/1.0 the package version was derived from the first three values in the assembly version, if no explicit value was given)

**Comment by danmosemsft** [comment]
@weshaggard can you review?

**Comment by Priya91** [comment]
Thanks @bartonjs

**Comment by Priya91** [comment]
```
D:\j\workspace\windows_nt_de---494cfe0d\Tools\Packaging.targets(954,5): error : System.Net.Security has mismatched compile (4.0.0.1) and runtime (4.0.0.2) versions on .NETCoreApp,Version=v1.0/win7-x86. [D:\j\workspace\windows_nt_de---494cfe0d\src\System.Net.Security\pkg\System.Net.Security.pkgproj]
```

@weshaggard @ericstj Can you please take a look at the changes.. Not sure what's causing the build error.

**Comment by bartonjs** [comment]
@Priya91 That's what the suppression file in #23232 is for.

Lock the assembly versions for any facades that are being built, then copy the suppressions file.

**Comment by Priya91** [comment]
Added the suppressionfile.

File: src/System.IO.FileSystem.DriveInfo/src/System.IO.FileSystem.DriveInfo.csproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
   <PropertyGroup>
     <RootNamespace>System.IO.FileSystem.DriveInfo</RootNamespace>
     <AssemblyName>System.IO.FileSystem.DriveInfo</AssemblyName>
-    <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <AssemblyVersion>4.0.0.1</AssemblyVersion>
```
Comment (bartonjs) [inline src/System.IO.FileSystem.DriveInfo/src/System.IO.FileSystem.DriveInfo.csproj]:
```xml
<AssemblyVersion Condition="'$(TargetGroup)' == 'net46'">4.0.0.0</AssemblyVersion>
```

Similar for the other files if they contain any TargetGroup which has a `IsPartialFacadeAssembly` value,

**Comment by danmosemsft** [comment]
replaced

**Pull request closed without merging**